### ISSUE TITLE
508-defect-2 [COGNITION]: Both forms' VA logo should be announced for authenticity

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26580]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Team PM will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear the VA logo image announced. This is important to help me know the survey is authentic and part of the VA.gov experience.

## Details

Right now the VA logo images have an empty alt attribute. We could change this by making the image an SVG for better scaling (won't get blurry) and add alt text.

## Acceptance Criteria

- [ ] Image announces to screen readers with meaningful alt text
- [ ] Ideally the image is an SVG so it scales on high-resolution screens

## Proposed Solution (if known)
Angular-specific code removed for brevity

```diff
<img 
- alt=""
+ alt="Department of Veterans Affairs logo"
  src="/path/to/va.svg"
/>
```

## Screenshots or Trace Logs

<img width="522" alt="Screen Shot 2021-06-24 at 12 15 03 PM" src="https://user-images.githubusercontent.com/934879/123306195-99a18c80-d4e6-11eb-8b31-0416d91473b8.png">
